packages/rn-tester/js/examples/FlatList/FlatList-onStartReached.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow
+ */
+
+'use strict';
+import type {RNTesterModuleExample} from '../../types/RNTesterTypes';
+import BaseFlatListExample, {ITEM_HEIGHT} from './BaseFlatListExample';
+import * as React from 'react';
+
+export function FlatList_onStartReached(): React.Node {
+  const [output, setOutput] = React.useState('');
+  const exampleProps = {
+    onStartReached: (info: {distanceFromStart: number, ...}) =>
```

Comment:
Is this called when a new list is mounted at zero position?

Reply:
Yes it will, I think both behaviour seem ok to me.